ci/travis/script-run-tests.sh

```diff
@@ -28,7 +28,11 @@ echo
 ${TRAVIS_BUILD_DIR}/scripts/run_nupic_tests.py -u --coverage --failfast || exit
 
 mv ${TRAVIS_BUILD_DIR}/.coverage ${TRAVIS_BUILD_DIR}/.coverage_unit
-# Python integration tests and prep for coveralls reporting
-${TRAVIS_BUILD_DIR}/scripts/run_nupic_tests.py -i --coverage --failfast || exit
 
-mv ${TRAVIS_BUILD_DIR}/.coverage ${TRAVIS_BUILD_DIR}/.coverage_integration
+# Removed because integration tests take so long and we run them in
```

Comment:
We removed integration tests from our builds like 2 years ago, mostly because the swarming tests are in there, and they take 20-30 minutes. I'll add them back in, but I think we should try to keep iterative build test phase under 10 minutes.